cdk/l2-cdk-resources/third-party-integration/.projenrc.js

```diff
@@ -0,0 +1,38 @@
+const { awscdk } = require('projen');
+const { ReleaseTrigger } = require('projen/lib/release');
+const thirdPartyIntegration = new awscdk.AwsCdkConstructLibrary({
+  author: 'MongoDBAtlas',
+  authorAddress: 'https://mongodb.com',
+  authorName: 'MongoDBAtlas',
+  cdkVersion: '2.1.0',
+  defaultReleaseBranch: 'INTMDB-548',
+  majorVersion: 1,
+  releaseToNpm: true,
+  releaseTrigger: ReleaseTrigger.manual(),
+  docgen: true,
+  name: '@mongodbatlas-awscdk/atlas-integrations',
+  repositoryUrl: 'https://github.com/mongodb/mongodbatlas-cloudformation-resources.git',
+  sampleCode: false,
+  keywords: ['cdk',
+    'awscdk',
+    'aws-cdk',
+    'cloudformation',
+    'cfn',
+    'extensions',
+    'constructs',
+    'cfn-resources',
+    'cloudformation-registry',
+    'l2',
+    'mongodb',
+    'atlas',
+    'third-party-integration'],
+  description: 'Retrieves or creates third-party-integrations with MicrosoftTeams, Datadog and PagerDuty for a given Atlas project',
```

Comment:
```suggestion
  description: 'Returns, adds, edits, and removes third-party service integration configurations. MongoDB Cloud sends alerts to each third-party service that you configure. Currently supporting MicrosoftTeams, Datadog and PagerDuty for a given Atlas project',
```

Reply:
Copied and then modified description from API docs: https://docs-atlas-dotcomstg.s3.us-east-2.amazonaws.com/docs-qa/atlas/reference/api-resources-spec/index.html#tag/Third-Party-Integrations

Reply:
Updated in follow-up PR: https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/311